html/images/os/allworx.svg

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
You need to remove the width/height + compress the svg.

Use https://jakearchibald.github.io/svgomg/ to do this.